app/assets/javascripts/discourse/app/templates/components/emoji-uploader.hbs

```diff
@@ -34,21 +34,21 @@
   </div>
   <div class="control-group">
     <div class="input">
-      <label class="btn btn-default btn-primary {{if addDisabled "disabled"}}">
+      <input
+        class="hidden-upload-field"
+        disabled={{uploading}}
+        type="file"
+        multiple="true"
+        accept=".png,.gif">
+      {{#d-button class="btn-primary" action=(action "chooseFiles") disabled=uploading}}
         {{#if uploading}}
-          {{d-icon "spinner" class="loading-icon"}}
+          {{d-icon "spinner" class="loading-icon"}} &nbsp;
```

Comment:
Curiously, do we usually use this or css? I tend to prefer css as I've seen `&nbsp;` cause issues for responsiveness before.

No strong opinions about it though.

Reply:
I will check, the previous button added this padding from the icon OK

Reply:
I fixed this, I didn't need to use the `yield` version of `d-icon`, it's all good now.